server/controllers/stock/core.js

```diff
@@ -233,6 +234,100 @@ function getLots(sqlQuery, parameters, finalClause = '', orderBy = '') {
   return db.exec(query, queryParameters);
 }
 
+/**
+ * Get asset info
+ *
+ * @param {string} depotUuid
+ * @param {object} params
+ */
+async function getAssets(params, finalClause) {
+
+  // NOTE: Expected parameters
+  //   depot_uuid, is_asset, scan_start_date, scan_end_date, reference_number
+  //   scan_status ==> one of 'all', 'scanned', or 'unscanned'
+
+  // Construct the WHERE clause to insert inside the join
+  // (Not obvious how to do this with filters since it is inside a JOIN)
+  let scanPeriod = '';
+  if ('scan_start_date' in params) {
+    const startDate = moment(params.scan_start_date).format('YYYY-MM-DD HH:MM:SS');
+    scanPeriod = `scan.created_at >= DATE('${startDate}')`;
```

Comment:
Does `startDate` come from the client?  If so, we should escape it with `db.escape()`.

Reply:
Did what you suggested.  However, I would suspect moment would barf first.